src/test/scala/org/constellation/domain/transaction/TransactionServiceTest.scala

```diff
@@ -398,7 +398,43 @@ class TransactionServiceTest
       txService.put(tx, ConsensusStatus.Unknown).unsafeRunSync
 
       val txs = txService.findByPredicate(_ => true).unsafeRunSync
-      txs isEmpty
+      txs shouldBe empty
```

Comment:
That line is not related to the main issue, but I just noticed that I left this error some time ago (`isEmpty` is not an assertion).

If you want I can move it to a different PR, please let me know.